src/methods/defining_methods/hard/q7/CurrentYearMethod.java

```diff
@@ -0,0 +1,31 @@
+package methods.defining_methods.hard.q7;
+
+/**
+ * [J] Write a Java Program that asks users for the number of days in the current year. Then write a
+ * method that accepts that value and returns a boolean value if it is a leap year or not (true if it
+ * is a leap year, false if not). Let’s call the method “leapChecker”.
```

Comment:
Change question text to: 
`Write a Java Program that asks users for the number of days in the current year. Then write a method called leapChecker that accepts that value as an integer and returns a boolean value if it is a leap year or not (true if it is, false if not). The year is a leap year if it has 366 days. Lastly, if the method returns true, print "This year is a leap year", if it is false, print "This year isn't a leap year". `